LEGO1/mxbitmap.h

```diff
@@ -1,13 +1,30 @@
 #ifndef MXBITMAP_H
 #define MXBITMAP_H
 
-class MxBitmap
+#include <stdlib.h>
+
+#include "legoinc.h"
+#include "mxcore.h"
+#include "mxpalette.h"
+
+class MxBitmap : public MxCore
 {
 public:
   __declspec(dllexport) MxBitmap();
   __declspec(dllexport) virtual ~MxBitmap();
   __declspec(dllexport) virtual MxPalette *CreatePalette();
-  __declspec(dllexport) virtual long Read(const char *);
+  __declspec(dllexport) virtual long Read(char const *);
+
+  const char *GetClassName() const;
+  MxBool IsClass(const char *name) const;
+  long Tickle();
```

Comment:
These are not present in the OG dll.
They just forward to the parent `MxCore`.

Reply:
so do these three functions need to be in the header?

Reply:
The errors showed up because I put in an extra instruction in isle.mak